Updated Blocks section (#244)
MinecraftForge/Documentation: docs/blocks/blocks.md

- `hardnessAndResistance` - The hardness controls the time it takes to break the block. It is an arbitrary value. For reference, stone has a hardness of 1.5, and dirt 0.5. If the block should be unbreakable a hardness of -1.0 should be used, see the definition of `bedrock` as an example. The resistance controls the explosion resistance of the block. For reference, stone has a resistance of 6.0, and dirt 0.5.
- `sound` - Controls the sound the block makes when it is punched, broken, or placed. Requires a `SoundType` argument, see the [sounds][] page for more details.
- `lightValue` - Controls the light emission of the block. Takes a value from zero to fifteen.
- `slipperiness` - Controls how slippery the block is. For reference, ice has a slipperiness of 0.98.

All these methods are *chainable* which means you can call them in series. See the `Blocks` class for examples of this.

!!! Note

>     Blocks have no setter for their `ItemGroup`. (formerly Creative Tab) This has been moved to the `BlockItem` and is now its responsibility. Furthermore, there is no setter for translation key. (this is generated based on registry name now)

Reviewer: Remove the parenthetical, or just note it more briefly
```suggestion
    Blocks have no setter for their `ItemGroup`. (formerly Creative Tab) This has been moved to the `BlockItem` and is now its responsibility. Furthermore, there is no setter for translation key as it is now generated from the registry name.
```
